OpenTSDB: Support v2.4 (#100673)
grafana/grafana: docs/sources/datasources/opentsdb/_index.md

While using OpenTSDB 2.2 data source, make sure you use either Filters or Tags as they are mutually exclusive. If used together, might give you weird results.
{{% /admonition %}}

{{% admonition type="note" %}}

> When using OpenTSDB 2.4 with alerting the query will be executed with the parameter `arrays=true`. This will return the data points from OpenTSDB as an array of arrays rather than a map of key-value pairs. Grafana will then convert this data into the appropriate data frame format.

Reviewer: ```suggestion
When using OpenTSDB 2.4 with alerting, queries are executed with the parameter `arrays=true`. This causes OpenTSDB to return data points as an array of arrays instead of a map of key-value pairs. Grafana then converts this data into the appropriate data frame format.
```

Reviewer: We want to use present tense as much as possible unless you are writing a tutorial, or it's clear something will happen in the future. https://grafana.com/docs/writers-toolkit/write/style-guide/style-conventions/#write-in-present-tense.

{{% /admonition %}}

### Auto complete suggestions

> As soon as you start typing metric names, tag names and tag values , you should see highlighted auto complete suggestions for them.

Reviewer: ```suggestion
As you begin typing metric names, tag names, or tag values, highlighted autocomplete suggestions will appear.
```
